packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -59,31 +58,19 @@ async function generateCertificate(
       );
       return thumbprint;
     } catch (err) {
-      console.log(
-        chalk.yellow(
-          'Failed to generate Self-signed certificate. Using Default Certificate. Use Visual Studio to renew it.',
-        ),
-      );
-      toCopyTempKey = true;
+      console.log(chalk.yellow('Failed to generate Self-signed certificate.'));
     }
-  } else {
-    console.log(
-      chalk.yellow('Using Default Certificate. Use Visual Studio to renew it.'),
-    );
-    toCopyTempKey = true;
-  }
-  if (toCopyTempKey) {
```

Comment:
Took me a second to understand why eslint was complaining about this, but it turns out the only case where we didn't assign this to false, we did an early return. Restructured this to accommodate that.